virt/vm_networking/virt-configuring-dpdk-workloads.adoc

```diff
@@ -0,0 +1,31 @@
+:_content-type: ASSEMBLY
+[id="virt-configuring-dpdk-workloads"]
+= Configuring Data Plane Development Kit workloads
```

Comment:
```suggestion
= Configuring cluster and VMs for DPDK workloads
```

Reply:
I have not heard anybody using the full name

Reply:
We have to spell out abbreviations in full. But if you think it is sufficiently well-known, then I can use the full name in the first sentence and the abbreviation in the title.